apps/e2e/forms-storybook/src/app/checkbox/checkbox.component.html

```diff
@@ -87,9 +88,21 @@
           [formGroup]="standardCheckboxGroupFormGroup"
           labelText="Contact method"
         >
-          <sky-checkbox formControlName="email" labelText="Email" />
-          <sky-checkbox formControlName="phone" labelText="Phone" />
-          <sky-checkbox formControlName="text" labelText="Text" />
+          <sky-checkbox
+            formControlName="email"
+            labelText="Email"
+            hintText="Hint text"
+          />
+          <sky-checkbox
+            formControlName="phone"
+            labelText="Phone"
+            hintText="Hint text"
```

Comment:
could we add an error under one of these hint texts within the group to make sure it works fine with the hint text inside a checkbox group?

Reply:
not really part of this story. I can look into adding that separately

Reply:
added

Reply:
in the visual test for this i see the required asterisk but i dont see the error showing up underneath, are we missing a click action with it?

Reply:
it was there, but it wasn't working for some reason. removing this change as it's effectively redundant with the other check